themes/src/main/resources/theme/keycloak.v2/login/register.ftl

```diff
@@ -73,22 +73,23 @@
             ].filter(p => p.policy.value !== -1);
 
             document.getElementById("password").addEventListener("change", (event) => {
-                const serverErrors = document.getElementById("input-error-password");
```

Comment:
I never really understood why this code exists, but could you elaborate on why it needs to be removed?

Reply:
Sure. When the registration form was submitted and the password didn't comply with the required password policies, the server would return an error message for the password. The element where the message was added into had the id `input-error-password`.

When the password verification with JavaScript was added, the author had to remove this error message, so it wouldn't be next to the error messages added by JavaScript. That's what this line of code does.

Since there's now only one error message div for both the server error message and for the JavaScript, it already get's removed by your `.replaceChildren(template)`.

Reply:
Got it thanks, consider this resolved.